### DISCUSSION TITLE
A lot of issues, package import, combo fields

### DISCUSSION THREAD

**Discussion by muneebirao** [joomengine/Joomla-Component-Builder#958] [Extention Issues] [answered]
Hi,

I am using JCB 3.1.24 with Joomla 3.10.12 and Php 7.4.28

I am unable to import ANY JCB package. Tried to import Sermon Distributor, Field Type Updater and others, with Valid keys, but nothing gets imported. It always says that import is successful, but nothing arrives in joomla.

Moreover, I am unable to use any Combo field type, Neither just basic combo, nor Calendar field. It always stores "0" in the database, no matter whatever value I select.

Please look into the issue. If there is anything wrong which I am doing, kindly point out. Thank you

**Comment by Llewellynvdm** [comment, marked as answer]
https://git.vdm.dev/joomla/Component-Builder/issues/1011
